frontend/src/lib/experimental/FeedbackCallCTA.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import './NPSPrompt.scss' // Lazy, but this is an experimental feature so not worth optimizing
+import { CloseOutlined } from '@ant-design/icons'
+import { Button } from 'antd'
+import { kea, useActions, useValues } from 'kea'
+import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
+import { FEATURE_FLAGS } from 'lib/constants'
+import { feedbackCallLogicType } from './FeedbackCallCTAType'
+import posthog from 'posthog-js'
+import { successToast } from 'lib/utils'
+
+const FEEDBACK_CALL_LOCALSTORAGE_KEY = 'call-cta-exp-2201'
+const APPEAR_TIMEOUT = 15000
+
+const COPY = [
+    {
+        title: 'Want to test product changes before shipping? 🧪',
+        description:
+            'Our new A/B testing feature is launching soon and will let you seamlessly run experiments on your product. Interested?',
+    },
+    {
+        title: 'Introducing A/B testing 🚀',
+        description:
+            'We are launching our A/B testing suite soon! Share your feedback, help us shape this new feature.',
+    },
+]
+
+const feedbackCallLogic = kea<feedbackCallLogicType>({
+    path: ['lib', 'experimental', 'FeedbackCallCTA'],
+    selectors: {
+        featureFlagGroup: [
+            () => [featureFlagLogic.selectors.featureFlags],
+            (featureFlags): null | 'control' | 'variant-0' | 'variant-1' =>
+                featureFlags[FEATURE_FLAGS.FEEDBACK_CALL_CTA] as null | 'control' | 'variant-0' | 'variant-1',
+        ],
+        copy: [
+            (s) => [s.featureFlagGroup],
+            (featureFlagGroup) =>
+                featureFlagGroup === 'variant-0' ? COPY[0] : featureFlagGroup === 'variant-1' ? COPY[1] : null,
+        ],
+    },
+    actions: {
+        reportAndDismiss: (result: 'call' | 'more-info' | 'not-interested' | 'dismiss') => ({ result }),
+        show: true,
+        hide: true,
+    },
+    reducers: {
+        hidden: [true, { show: () => false, hide: () => true }],
+    },
+    listeners: ({ actions }) => ({
+        reportAndDismiss: ({ result }) => {
+            posthog.capture('experimentation call prompt action', { result })
+            localStorage.setItem(FEEDBACK_CALL_LOCALSTORAGE_KEY, 'true')
```

Comment:
Optional nit: We have kea-localstorage for this! https://keajs.org/docs/plugins/localstorage - worth using it for the hidden reducer?

And then wherever you use localStorage.getItem, we can instead use `values.hidden`.

Reply:
I'm aware of the kea plugin, opted to do this because it's just a hacky temporary thing, but will use this next time.